spec/lib/airbrake_api/v3/notice_parser_spec.rb

```diff
@@ -41,33 +41,8 @@
     )
     expect(notice.session).to include('isAdmin' => true)
     expect(notice.params).to include('returnTo' => 'dashboard')
-    request_env = {}
     expect(notice.env_vars).to include(
-      'navigator_vendor' => 'Google Inc.',
-      'HTTP_ACCEPT' => request_env['HTTP_ACCEPT'],
-      'HTTP_ACCEPT_ENCODING' => request_env['HTTP_ACCEPT_ENCODING'],
-      'HTTP_ACCEPT_LANGUAGE' => request_env['HTTP_ACCEPT_LANGUAGE'],
-      'HTTP_CACHE_CONTROL' => request_env['HTTP_CACHE_CONTROL'],
-      'HTTP_CONNECTION' => request_env['HTTP_CONNECTION'],
-      'HTTP_COOKIE' => request_env['HTTP_COOKIE'],
-      'HTTP_HOST' => request_env['HTTP_HOST'],
-      'HTTP_IF_NONE_MATCH' => request_env['HTTP_IF_NONE_MATCH'],
-      'HTTP_UPGRADE_INSECURE_REQUESTS' => request_env['HTTP_UPGRADE_INSECURE_REQUESTS'],
-      'HTTP_USER_AGENT' => request_env['HTTP_USER_AGENT'],
-      'HTTP_VERSION' => request_env['HTTP_VERSION'],
-      'HTTP_X_AMZ_SERVER_SIDE_ENCRYPTIO' => request_env['HTTP_X_AMZ_SERVER_SIDE_ENCRYPTIO'],
-      'ORIGINAL_FULLPATH' => request_env['ORIGINAL_FULLPATH'],
-      'ORIGINAL_SCRIPT_NAME' => request_env['ORIGINAL_SCRIPT_NAME'],
-      'PATH_INFO' => request_env['PATH_INFO'],
-      'QUERY_STRING' => request_env['QUERY_STRING'],
-      'REMOTE_ADDR' => request_env['REMOTE_ADDR'],
-      'REQUEST_METHOD' => request_env['REQUEST_METHOD'],
-      'REQUEST_PATH' => request_env['REQUEST_PATH'],
-      'REQUEST_URI' => request_env['REQUEST_URI'],
-      'SERVER_NAME' => request_env['SERVER_NAME'],
-      'SERVER_PORT' => request_env['SERVER_PORT'],
-      'SERVER_PROTOCOL' => request_env['SERVER_PROTOCOL'],
-      'SERVER_SOFTWARE' => request_env['SERVER_SOFTWARE']
+      'navigator_vendor' => 'Google Inc.'
```

Comment:
This is just reverting to original code: https://github.com/errbit/errbit/blob/master/lib/airbrake_api/v3/notice_parser.rb#L57
original spec: https://github.com/errbit/errbit/blob/master/spec/lib/airbrake_api/v3/notice_parser_spec.rb#L45